composer/algorithms/fused_layernorm/fused_layernorm.py

```diff
@@ -0,0 +1,85 @@
+# Copyright 2022 MosaicML Composer authors
+# SPDX-License-Identifier: Apache-2.0
+
+# Copyright 2022 MosaicML. All Rights Reserved.
+
+from __future__ import annotations
+
+import logging
+from typing import Dict, Optional, Sequence, Type, Union
+
+import torch
+
+try:
+    from apex.normalization.fused_layer_norm import FusedLayerNorm as APEXFusedLayerNorm
+    APEX_INSTALLED = True
+except ImportError as e:
+    APEX_INSTALLED = False
+
+from composer.core import Algorithm, Event, State
+from composer.loggers import Logger
+from composer.utils import module_surgery
+
+log = logging.getLogger(__name__)
+
+
+def from_LayerNorm(layer: torch.nn.Module, module_index: int) -> APEXFusedLayerNorm:
+    """Defines a replacement policy from a `torch.nn.LayerNorm` to a `apex.normalization.fused_layer_norm`"""
+    assert isinstance(layer,
+                      torch.nn.LayerNorm), 'The replacement policy will look for all instances of torch.nn.LayerNorm'
+    return APEXFusedLayerNorm(normalized_shape=layer.normalized_shape, eps=layer.eps)
+
+
+def apply_fused_layernorm(model: torch.nn.Module, optimizers: Union[torch.optim.Optimizer,
+                                                                    Sequence[torch.optim.Optimizer]]) -> None:
+    """Replaces all instances of `torch.nn.LayerNorm` with a `apex.normalization.fused_layer_norm.FusedLayerNorm
+    <https://nvidia.github.io/apex/layernorm.html>`_.
+
+    By fusing multiple kernel launches into one, this usually improves GPU utilization.
+    """
+    if not APEX_INSTALLED:
+        raise ImportError(
+            'https://github.com/NVIDIA/apex is not installed. The Fused LayerNorm algorithm cannot be applied.')
```

Comment:
Could these errors be refactored out into a shared helper? I see the message is repeated below.